R/get_summary_records.R

```diff
@@ -64,9 +71,9 @@
 #'
 #' @return A data frame of derived records.
 #'
-#' @family der_gen
+#' @family superseded
 #'
-#' @keywords der_gen
+#' @keywords internal
```

Comment:
I made a superseded section on the reference page.

```suggestion
#' @keywords superseded
```

Reply:
fixed